### DISCUSSION TITLE
bootable flashes over 64-128GB won't detected on some old laptops,systems

### DISCUSSION THREAD

**Discussion by amirbahalegharn** [ventoy/Ventoy#1818] [General]
hi
can ventoy do sth about it ? because there are laptops (& systems) which won't be able to boot from USB devices with storage above 64 or 128GB, so can ventoy change sth to enforce reading/detecting it as a 8GB in that first boot sector or whatever to bypass it like some of those fake chinese usb flashes with intentional miscalculated storage size? because of this problem, i always have to carry two usb flash,one 8GB and another 256gb one for all my portable storage needs. it's specially important as years goes by and the size of files for softwares,drivers,etc all are increasing and thus the needs & using larger usb storage much more common sense for more and more people and unfortunately some old laptops & system(even from a decade ago) show issue booting from large usb media and it would be great if ventoy developer could address that as well if possible. 
thanks in advance

**Comment by a1ive** [comment]
https://www.ventoy.net/en/doc_legacy_limit.html
